src/utils/arrayFunctions.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Gets the uniq elements from giver array
+ * @param array The array object
+ */
+export const uniqueArray = <T>(array: T[]): T[] => {
```

Comment:
function isn't exported correctly + file name should match function name

Reply:
@TQjona ,  file name updated and fixed export default